### PR TITLE
[IMP] website: introduce `s_wavy_grid` snippet

### PR DESCRIPTION
**Pull request by Brieuc-brd**
This commit adds the new `s_wavy_grid` snippet.

task-4105125
Part of task-4077427

Requires:
- https://github.com/odoo/design-themes/pull/883

67255544-master-design_theme runbot178 odoo com_

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/website/views/snippets/snippets.xml [hunk 1]
```diff
@@ -87,6 +87,9 @@
                 <t t-snippet="website.s_three_columns" string="Columns" group="columns">
                     <keywords>columns, description</keywords>
                 </t>
+                <t t-snippet="website.s_wavy_grid" string="Wavy Grid" group="columns">
```
Comment (chgo-odoo) [inline addons/website/views/snippets/snippets.xml:90]: As discussed, don't forget to rebase and move the declaration of your snippet [to the right place](https://github.com/odoo/odoo/blob/master/addons/website/views/snippets/snippets.xml#L65)

**Review by anso-odoo — APPROVED** [review]
LGTM 👍

**Comment by Brieuc-brd** [comment]
Fixed heading display class: https://github.com/odoo/odoo/compare/b94fc4dcaffc69c88e84d9c5406477708bd5069b..de2950206e7c5e4561a0afa6fe7f605ce1acac9d

File: addons/website/views/snippets/s_wavy_grid.xml [hunk 2]
```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_wavy_grid" name="Wavy Grid">
+    <section class="s_wavy_grid pt32 pb32" data-oe-shape-data="{'shape':'web_editor/Bold/06_001','flip':[],'showOnMobile':false,'shapeAnimationSpeed':'0'}">
+        <div class="o_we_shape o_web_editor_Bold_06_001"/>
+        <div class="container">
+            <div class="row o_grid_mode" data-row-count="29">
```
Comment (bvr-odoo) [inline addons/website/views/snippets/s_wavy_grid.xml:8]: I count 30 rows

File: addons/website/models/website.py [hunk 3]
```diff
@@ -852,6 +852,11 @@ def fallback_create_missing_industry_image(image_name, fallback_img_name):
             fallback_create_missing_industry_image('s_text_cover_default_image', 's_cover_default_image')
             fallback_create_missing_industry_image('s_showcase_default_image', 's_image_text_default_image')
             fallback_create_missing_industry_image('s_accordion_image_default_image', 's_image_text_default_image')
+            fallback_create_missing_industry_image('s_image_title_default_image', 's_cover_default_image')
```
Comment (bvr-odoo) [inline addons/website/models/website.py:855]: Why you add this line ? It seems related to the `s_image_title` snippet but I don't know/find this snippet

**Review by bvr-odoo — CHANGES_REQUESTED** [review]
@Brieuc-brd Nice snippet :+1: , here is my review :wink:

File: addons/website/models/website.py (hunk 3 above)
Comment (Brieuc-brd) [reply addons/website/models/website.py]: Oops, I forgot to remove this leftover, thanks !

**Comment by Brieuc-brd** [comment]
Thanks for your review @bvr-odoo 👍 
I've updated the branches, could you have a look ? 
The diffs : 
- COM: https://github.com/odoo/odoo/compare/de2950206e7c5e4561a0afa6fe7f605ce1acac9d..ae9b46a47f39c0a378a0eed1eb7cf69e6b4605b8
- DT: https://github.com/odoo/design-themes/compare/a4190098c26606f643c6009642aead07897980ce..96e0c2d240860ab854c7609ad7410e833a45f852

**Review by bvr-odoo — APPROVED** [review]
@Brieuc-brd Thanks for the changes :+1:

**Pull request closed without merging**